materialize-snowflake/snowflake.go

```diff
@@ -88,12 +96,14 @@ func (c *config) Validate() error {
 			return fmt.Errorf("missing '%s'", req[0])
 		}
 	}
+
+	if !validHost(c.Host) {
+		return fmt.Errorf("invalid host %q (must match end in snowflakecomputing.com and not include a protocol)", c.Host)
```

Comment:
Nit: I think it's better if `validHost` itself gives this error, since that function knows what it is checking, I would expect the caller to not have to know what is happening inside the function